packages/client/src/pages/Home/components/Rules/Rules.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FC } from 'react'
+import style from './rules.module.css'
+import { clsx } from 'clsx'
+import { RulesProps } from './types'
+import Divider from '../../../../components/Divider/Divider'
+
+const Rules: FC<RulesProps> = ({ className }) => {
+  return (
+    <div className={clsx(style.containerRules, className)}>
+      <div className={style.dividers}>
+        <Divider className={style.shortDivider} />
+        <Divider className={style.longDivider} />
+      </div>
+      <div className={style.content}>
+        <div className={style.titlesContainer}>
+          <div className={style.subtitle}>Об игре</div>
+          <div className={style.title}>Как играть?</div>
+        </div>
+        <div className={style.about}>
+          <div className={style.startImage}>
+            <div className={style.aboutImage}>
+              <img src="../../../../../public/playStart.png" alt="play" />
```

Comment:
Картинки лучше импортировать из `@assets/img`. Так безопаснее
```suggestion
              <img src={PlayStartImg} alt="play" />
```